moj-node/server/utils/index.js

```diff
@@ -39,10 +39,21 @@ function fixUrlForProduction(url, alternateUrl = config.hubEndpoint) {
   return url;
 }
 
+function getEventTitle(event) {
+  const fixedTitle = event.eventLocation
```

Comment:
What do you mean by `fixedTitle`?

Reply:
It's always going to be this title or this this title with a comment appended.

Reply:
I think just calling it title will be fine here?

Reply:
👍